about/list_of_features.rst

```diff
@@ -0,0 +1,510 @@
+.. _doc_list_of_features:
+
+List of features
+================
+
+This page aims to be a synthetic list of all features currently supported by Godot.
```

Comment:
I think the word synthetic here is lost in translation. I am having trouble understanding the difference between a list and a synthetic list.